### PR TITLE
allow eval script on Android

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by MorrisJobke** [comment]
😭 😭 😭

**Comment by MorrisJobke** [comment]
Are you sure there is no better way around it?

**Comment by tobiasKaminsky** [comment]
Sorry, some more background:
Android docker image is only used for testing.
eval is needed to inject javascript into a webview so that we can do UI tests on it.
This is needed to perform a real login via weblogin flow (within the webview).

Acoording to Android this is the only way to test webview.

**Comment by MorrisJobke** [comment]
Okay ... but it might break.

File: server/init.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ git checkout ${BRANCH}
 git pull
 git submodule update
 
+# allow eval script for executing javascript in webview (LoginIT test)
```
Comment (skjnldsv) [inline server/init.sh]:
```suggestion
# allow eval script for executing javascript in webview (LoginIT test)
if test -z "$EVAL"
then
	echo "\$EVAL not set, ignoring..."
else
	sed -i s'/protected $evalScriptAllowed = false;/protected $evalScriptAllowed = true;/' 		 
	lib/public/AppFramework/Http/ContentSecurityPolicy.php
fi
```

Comment (tobiasKaminsky) [reply server/init.sh]: Where would I have to set "$EVAL"? In drone?

Comment (skjnldsv) [reply server/init.sh]: in the environment section of the drone job :)

Comment (tobiasKaminsky) [reply server/init.sh]: Then I'll do this tomorrow.
Very good approach!

**Pull request merged**